app/models/health/agency_performance.rb

```diff
@@ -234,5 +247,12 @@ def qualifying_activity_dates
         pluck(:patient_id, :date_of_activity).to_h
     end
 
+    def qa_signature_dates
+      @qa_signatures ||= Health::QualifyingActivity.submittable.
+        where(patient_id: patient_referrals.keys). # limit to patients in scope
+        where(activity: :pctp_signed).
+        group(:patient_id).maximum(:date_of_activity)
```

Comment:
You lost the range.  Is that not necessary any more?

Reply:
Oops -- yeah, I think that should still be there...